daemon/events/metrics.go

```diff
@@ -0,0 +1,21 @@
+package events
+
+import (
+	"github.com/docker/docker/dockerversion"
+	"github.com/docker/go-metrics"
+)
+
+var (
+	eventsCounter    metrics.Counter
+	eventSubscribers metrics.Gauge
+)
+
+func init() {
+	ns := metrics.NewNamespace("engine", "events", metrics.Labels{
+		"version": dockerversion.Version,
```

Comment:
Here are the relevant nuggets from the Robustperception tech blog:

http://www.robustperception.io/how-to-have-labels-for-machine-roles/
http://www.robustperception.io/target-labels-are-for-life-not-just-for-christmas/
and in particular
http://www.robustperception.io/exposing-the-software-version-to-prometheus/